### PR TITLE
LPS-175809 remove ObjectEntries when a site is deleted

### PR DESCRIPTION
**Pull request by jgarcialr**
https://issues.liferay.com/browse/LPS-175809

### CODE REVIEW

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/model/listener/GroupModelListener.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+					}
+					catch (PortalException portalException) {
+						_log.error(portalException);
+					}
+				}
+			}
+		}
+		catch (PortalException portalException) {
+			_log.error(portalException);
+		}
```
Comment (carolmariaabb) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/model/listener/GroupModelListener.java:65]: @jgarcialr could you please include the following commits from the previous pull request?

https://github.com/brianchandotcom/liferay-portal/commit/e3107c0419c9ef341053253c73fc482ac2e62245 and https://github.com/brianchandotcom/liferay-portal/commit/2398462aaa3ba3c645db8d72ff42dc7dac9fb1bb

Thanks!

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/model/listener/GroupModelListener.java (hunk 1 above)
Comment (jgarcialr) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/internal/model/listener/GroupModelListener.java]: Oh, I missed those changes.
I'll add those to the PR.

**Comment by jgarcialr** [comment]
ci:test:object

**Comment by jgarcialr** [comment]
ci:test:relevant

**Comment by jgarcialr** [comment]
ci:test:sf

**Comment by jgarcialr** [comment]
ci:test:sf

**Comment by jgarcialr** [comment]
ci:test:sf

**Comment by jgarcialr** [comment]
ci:test:relevant

**Comment by jgarcialr** [comment]
ci:test:objects

**Comment by jgarcialr** [comment]
ci:test:object

**Review by carolmariaabb — APPROVED** [review]
LGTM!

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**